### ISSUE TITLE
Rule: vue/sort-props

### ISSUE THREAD

**Issue by jpickwell** [vuejs/eslint-plugin-vue#601]
**Please describe what the rule should do:**

Allow enforcing a sort order for props.

**What category of rule is this? (place an "X" next to just one item)**

[X] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js
/* eslint "vue/sort-props": [2. "asc"] */
{
  props: {
    required: Boolean,
    disabled: Boolean // error
  }
}
```

```js
/* eslint "vue/sort-props": [2. "desc"] */
{
  props: {
    disabled: Boolean,
    required: Boolean // error
  }
}
```

**Comment by armano2** [comment]
duplicate #541

**Comment by jpickwell** [comment]
@armano2, that one is talking about sorting the keys inside a prop; i'm talking about just sorting the props themselves

**Comment by armano2** [comment]
@jpickwell ahh well i got confused a little sorry 👍

**Comment by jpickwell** [comment]
Understandable.

**Comment by loren138** [comment]
Closed by #997

**Comment by ota-meshi** [comment]
Thank you @loren138!
